docs/ReleaseNotes.md

```diff
@@ -1,6 +1,42 @@
 # Acuminator Release Notes
 This document provides information about fixes, enhancements, and key features that are available in Acuminator.
 
+## Acuminator 2.0
+Acuminator 2.0 includes the bug fixes and enhancements described in this section, as well as the features that have been implemented in previous versions.
+
+### Key Changes
+Acuminator 2.0 includes the following key changes:
+
+ - Acuminator no longer supports Visual Studio 2015. 
+ - Acuminator now supports C# 7.0 and 8.0.
+
+### Enhancements to Suppression Mechanism
+With the new version of Acuminator, you can suppress a particular diagnostic in a specific place in your project as follows:
+
+ 1. Click the code that is highlighted by the Acuminator diagnostic so that the editor cursor is in the highlighted element.
+ 2. Click **Suppress Acuminator Diagnostic > in Acuminator Suppression File** on the **Acuminator** main menu of Visual Studio.
+
+The suppressed diagnostic is saved in the Acuminator suppression file, which is located in the folder of your Visual Studio project. This file has the same name as the project file and the `acuminator` extension. If there is no Acuminator suppression file in the project folder, the file is created automatically.
+
+Acuminator suppression mechanism differs from the standard suppression mechanism of Visual Studio. With the Acuminator suppression mechanism, you can suppress a particular diagnostic in a specific place in the project, while the standard mechanism suppresses all diagnostics with this ID in the method.
```

Comment:
@k-s-popova  it's not quite right. This sentence sounds like the standard mechanism cannot suppress diagnostic on property or field. The main difference is in granularity of suppression - the VS suppresion's minimum unit is a type member. You can suppress diagnostic on a type or on some of its members (or use pragma directive which looks ugly in code). The Acuminator suppression minimum unit is smaller - it's a syntax node. This allows to suppress diagnostic on a concrete statement.

Reply:
fixed